source/client-dependencies.txt

```diff
@@ -1,18 +1,271 @@
 .. _client-authentication:
 
-======================
-Authentication Plugins
-======================
+==============
+Authentication
+==============
 
 .. default-domain:: mongodb
 
-The |bi| offers authentication plugins allowing you to authenticate
-using either the ``SCRAM-SHA-1``, ``SCRAM-SHA-256``, or ``PLAIN``
-authentication mechanisms. If you are using the
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The |bi| supports the following authentication mechanisms for
+authenticating with MongoDB and for authenticating connecting users:
+
+- ``SCRAM-SHA-1``
+- ``SCRAM-SHA-256``
+- ``PLAIN`` (LDAP)
+- ``GSSAPI`` (Kerberos)
+
+The |bi-short| (:binary:`~bin.mongosqld`) uses admin credentials to
+authenticate with MongoDB and :ref:`sample data <bi-cached-sampling>` to
+generate the |bi-short|'s schema. Once authenticated,
```

Comment:
Perhaps replace `Once authenticated` with `When a client connects to the |bi-short|`. I find it easier to think about admin auth and user auth as two entirely separate things, and this phrase makes them sound like parts of one unified auth flow, which may be confusing.